File: src/exemplos/04_Subalgoritmos/modulos/subalgoritmos.c
/**      @file: subalgoritmos.c
 *     @author: Guilherme N. Ramos (gnramos@unb.br)
 * @disciplina: Algoritmos e Programação de Computadores
 *
 * Exemplo de uso da biblioteca com funcionalidades. */

#include "apc.h" /* Inclui a biblioteca. */

int main() {
  int i;

  printf("Agora é a hora: ");
  mostra_agora(); /* Função da biblioteca apc.h. */

  gera_numeros_aleatorios(); /* Função da apc.h. */

  for(i = 0; i < 5; ++i) {
    printf("Um número qualquer: %d\n", qualquer());     /* Função da apc.h */
    printf("Outro em [0,100):   %d\n", aleatorio(100)); /* Função da apc.h */
    printf("E mais um em [0,5): %d\n", aleatorio(5));   /* Função da apc.h */
  }

  return 0;
}

File: src/exemplos/04_Subalgoritmos/modulos/subalgoritmos.py
#  -*- coding: utf-8 -*-
#    @package: subalgoritmos.py
#     @author: Guilherme N. Ramos (gnramos@unb.br)
# @disciplina: Algoritmos e Programação de Computadores
#
# Exemplo de uso do módulo com funcionalidades.


import apc  # Inclui o módulo

print('Agora é a hora:', apc.mostra_agora())  # Função do módulo

for x in range(5):
    print('  Um número qualquer:', apc.qualquer())      # Função do módulo
    print('  Outro em [0,100):  ', apc.aleatorio(100))  # Função do módulo
    print('  E mais um em [0,5):', apc.aleatorio(5))
